server_admin/topics/clients/client-policies.adoc

```diff
@@ -0,0 +1,159 @@
+[[_client_policies]]
+=== Client Policies
+
+To make it easy to secure client applications, it is beneficial to realize the following points in a unified way.
+
+* Setting policies on what configuration a client can have
+* Validation of client configurations
+* Conformance to a required security standards and profiles such as FAPI
+
+To realize these points in a unified way, _Client Policies_ concept is introduced.
+
+==== Use-cases
+
+Client Policies realize the following points mentioned as follows.
+
+Setting policies on what configuration a client can have::
+    Configuration settings on the client can be enforced by client policies during client creation/update, but also during OpenID Connect requests to {project_name} server, which are related to particular client.
+    The {project_name} supports similar thing also through the Client Registration Policies described in the link:{adapterguide_link}#_client_registration_policies[{adapterguide_name}].
+    However, Client Registration Policies can only cover OIDC Dynamic Client Registration. Client Policies cover not only what Client Registration Policies can do, but other client
+    registration and configuration ways. In the future, Client Registration will be replaced by Client Policies in the future.
```

Comment:
```suggestion
    registration and configuration ways. In the future, Client Registration will be replaced by Client Policies.
```

Reply:
This is worded in a way that we are making a commitment to what a future release will contain.  With changing priorities and people being overscheduled, is this definite?
How about "The current plans are for Client Registration to be replace by Client Policies."

Reply:
@vmuzikar I am changing this in a way, which @andymunro proposed. Thanks for the suggestions!